src/main/java/org/stuartgunter/maven/plugins/couchbase/CreateBucketMojo.java

```diff
@@ -61,10 +63,22 @@
     @Parameter(required = true)
     private String proxyPort;
 
+    /**
+     * Path to properties file, that contains template of bucket properties
+     *
+     * @since 1.0.2
+     */
+    @Parameter
+    private String defaultPropertiesPath;
```

Comment:
I don't think this should be called `defaultPropertiesPath` given that the user can override it. Perhaps `propertiesPath`? Or even better, provide it as a `File` instead of a `String`?